executor/executor_ddl.go

```diff
@@ -72,6 +72,11 @@ func (e *DDLExec) Next() (*Row, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	// Update InfoSchema in TxnCtx, so it will pass schema check.
```

Comment:
After DDL execution, the session is not in transaction, so it's safe to update schema version.